code/game/objects/random.dm

```diff
@@ -48,20 +48,28 @@
 	name = "DO NOT USE"
 	desc = "This is a random gun."
 	icon_state = "dice"
+/*
+/obj/random/gun/random_ak
+	name = "random ak"
+	desc = "This is a random ak."
+	icon_state = "dice"
+	var/gun = pick(/obj/random/gun/ak47,/obj/random/gun/ak74,/obj/random/gun/ak_modern)
+/obj/random/gun/random_ak/New()
```

Comment:
The New() proc isn't functional, assuming you'll complete it later.

Reply:
Yep was just testing out some stuff